src/foldertreedelegateeditor.cpp

```diff
@@ -69,16 +101,44 @@ FolderTreeDelegateEditor::FolderTreeDelegateEditor(QTreeView *view,
     connect(dynamic_cast<NodeTreeView *>(m_view), &NodeTreeView::renameFolderRequested, m_label,
             &LabelEditType::openEditor);
     layout->addWidget(m_label);
+    layout->addSpacing(5);
     m_contextButton = new PushButtonType(parent);
     m_contextButton->setMaximumSize({ 33, 25 });
     m_contextButton->setMinimumSize({ 33, 25 });
     m_contextButton->setCursor(QCursor(Qt::PointingHandCursor));
     m_contextButton->setFocusPolicy(Qt::TabFocus);
-    m_contextButton->setIconSize(QSize(16, 16));
-    m_contextButton->setStyleSheet(QStringLiteral(R"(QPushButton { )"
-                                                  R"(    border: none; )"
-                                                  R"(    padding: 0px; )"
-                                                  R"(})"));
+    if (m_view->selectionModel()->isSelected(m_index)) {
+        m_contextButton->setStyleSheet(QStringLiteral(R"(QPushButton { )"
```

Comment:
Wouldn't it be possible to move all the styling set with `setStyleSheet()` to the CSS files we now have?

Reply:
You're right! I'll do that.

Reply:
I'm actually unsure how to go about that since these buttons are created synamically and not in the .ui file, so how can I select them via .css? @zjeffer

Reply:
I haven't found a way to do this yet with these dynamically created buttons. Same problem with the QPainter: as far as I can tell, there is no way to set the colors through CSS.

Reply:
Gotcha. So let's keep it as it is?